packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -66,10 +79,9 @@ type Props = {
   roots: Array<Node>,
   disableWrap: boolean,
   dimTopLevelWindow: boolean,
-  getObjectEntries: (actor:string) => ?LoadedEntries,
-  getObjectProperties: (actor:string) => ?LoadedProperties,
-  loadObjectEntries: (value:RdpGrip) => void,
-  loadObjectProperties: (value:RdpGrip) => void,
+  releaseActor: string => void,
```

Comment:
There are two types of `releaseActor` here.  Is this intentional, and does flow let you collapse this into one line? Or if not, can you move them to adjacent lines?

Reply:
not wanted, it should be only the one taking a string as an argument